test/supertest.js

```diff
@@ -150,7 +154,7 @@ describe('request(app)', function(){
 
     request(app)
     .get('/')
-    .expect(302, done);
+    .expect(302, function() { done(); });
```

Comment:
Thanks for pointing that out.  Seems like the changes to superagent may require more significant changes to the supertest's API to make them sensible and work well together.